### ISSUE TITLE
Input size

### ISSUE THREAD

**Issue by Dev2022** [dlyldxwl/fssd.pytorch#4]
Hi...Is the pruning for 300X300 only? I didnt find any pruning for 512X512. Thanks

**Comment by dlyldxwl** [comment]
@Dev2022 yes, I only have one titan x card so that I can't training the 512*512 model (It will affect the accuracy that batch_size is to small.)
